templates/toil-cluster-role-and-policy.yaml

```diff
@@ -49,6 +50,7 @@ Resources:
               - ec2:DescribeSecurityGroups
               - ec2:DescribeSpotInstanceRequests
               - ec2:DescribeSpotPriceHistory
+              - ec2:DescribeTags
```

Comment:
should this be `ec2:Describe*` instead? https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/essentials.yaml#L346

Reply:
If it were to match that role exactly. I can change it to this.